src/dumpContent.ts

```diff
@@ -88,6 +95,15 @@ function dumpManifest(objIds: string[]) {
   )
 }
 
+function preprocessObjData(objData: ObjData): ObjData {
+  // Ignore pisa_url - this should be set up in the new instance
```

Comment:
* [ ] FYI I think readers of this comment are not necessarily people setting up instances. Remove?